webpay/pay/forms.py

```diff
@@ -92,7 +92,7 @@ def clean_req(self):
             raise forms.ValidationError(msg.BAD_ICON_KEY)
 
         for fn in msg.SHORT_FIELDS:
-            if (len(payload['request'].get(fn, '')) >
+            if (len(payload['request'].get(fn, None) or '') >
```

Comment:
I found this hard to parse for a moment - but I see now. A comment might be handy although the test will avoid this getting accidentally "factored" out. :)